support/index.js

```diff
@@ -25,7 +25,7 @@ Cypress.on('window:before:load', win => {
 });
 
 before(async () => {
-  if (!Cypress.env('SKIP_METAMASK_SETUP')) {
+  if (Cypress.env('EXTENSION') === 'metamask' && !Cypress.env('SKIP_METAMASK_SETUP')) {
     await cy.setupMetamask();
   }
```

Comment:
we should probably add a case for keplr here as well

Reply:
At the moment, I have to implement the downloading logic for Keplr. Once that's done, I'll adjust it